### ISSUE TITLE
Behat outdated

### ISSUE THREAD

**Issue by cauerego** [ari-gold/WordPress-Behat-Quickstart#1]
Following https://github.com/Behat/Behat/issues/689#issuecomment-144349592 I realised behat is outdated.

On composer you've forced Behat 2.4 and Mink 1.4. Any reason why? How should we proceed to update this?

**Comment by ari-gold** [comment]
Thanks for reporting this issue @cauerego! 

I'll test upgrading Behat and Mink and report back.

**Comment by cauerego** [comment]
Oh... Meta BDD! :+1:

**Comment by ari-gold** [comment]
HI @cauerego

I submitted an issue to the Behat docs: https://github.com/Behat/docs/issues/90

In the meantime, it should be safe to ignore that deprecation notice. I just followed the guide (https://pantheon.io/docs/guides/automated-testing-wordpress-behat/) again on a new WordPress install again and it seemed to work fine.

**Comment by cauerego** [comment]
Yeah, my installation didn't seen to have any issues either, but me being new to all this as well I couldn't be sure.

And cool, now I see where you got the version from! :)

At this point, wouldn't it be better to update it to use 3 though? Asked about the same question there as well.
